tests/test_generators.py

```diff
@@ -7,11 +7,11 @@
 from .util import run_in_separate_process, xfail_on_ci_if_model_is_incompatible
 
 # generate list of all models in impy.models
-models = set(obj for obj in im.__dict__.values() if type(obj) is abc.ABCMeta)
+models = list(obj for obj in im.__dict__.values() if type(obj) is abc.ABCMeta)
 
 
 def run_model(model, ekin):
-    gen = model(ekin)
+    gen = model(ekin, seed=349745)
```

Comment:
There is no need to use a complicated seed, it is better to use something simple like seed=1.

Reply:
I will change that to `seed=1`. I hope that it is not a special value.

Reply:
Ok, I see the point. If this generator follows any conventions, then the only special value is 0. Looking into rangen.f, that seems to be the case, there is special case, so using 1 is ok.